images.go

```diff
@@ -0,0 +1,52 @@
+package shopify
+
+import "time"
+
+// Images is a collection of Images
+type Images []Image
+
+// Image is an image used by the product
+type Image struct {
```

Comment:
This should be called a product image as it has fields on it that are specific to products